src/sentry/lang/native/utils.py

```diff
@@ -61,23 +61,7 @@ def cpu_name_from_data(data):
     if device and device.get('arch'):
         return device['arch']
 
-    # TODO: kill this here.  we want to not support that going forward
-    unique_cpu_name = None
```

Comment:
How risky do you think this is? Would it make sense to deploy this separately or does the old code already depend on old symbolic?

Reply:
I've checked this, and this has long been dead code. The Cocoa SDK (the only one where this would matter) has been sending an `arch` forever. I definitely do **not** want this in a separate PR.